src/components/LeftNavigation/LeftNavigation.Props.ts

```diff
@@ -24,6 +24,7 @@ export interface ILeftNavigationOption {
     selected?: boolean;
     disabled?: boolean;
     position?: LeftNavigationOptionPositionEnum;
+    notificationNumber?: number;
```

Comment:
I'd like to see a specialized version of Icon, called NotificationIcon or some kind of CompositeIcon which can combine multiple SVGs. That way we can have Icons with "NEW" text or something similar, and also combine two Icons programatically if there is need?
I feel like this notificationNumber is not a part of generic Icon component..

Reply:
I removed NotificationIcon from Icon component. NotificationIcon is now a separate component.